K8s-deployment/Charts/consent-validator/example-secrets/secrets/config.json

```
@@ -0,0 +1,46 @@
+{
+    "version": "1.0",
+    "postgresOptions": {
+      "databaseIP": "database.iudx.io",
```

```suggestion
      "databaseIP": "psql-postgresql-ha-pgpool.postgres.svc.cluster.local",
```
Use postgres' service